src/api/resources/index.ts

```diff
@@ -330,6 +491,77 @@ export class ResourcesApi {
       res.json(this.getResourcePaths())
     })
 
+    // Providers: Return list of providers
+    server.get(
+      `${RESOURCES_API_PATH}/:resourceType/_providers`,
+      async (req: Request, res: Response) => {
+        debug(`** ${req.method} ${req.path}`)
+        res.json(this.getProvidersForResourceType(req.params.resourceType))
+      }
+    )
+
+    // Providers: Return the default provider for the supplied resource type
+    server.get(
+      `${RESOURCES_API_PATH}/:resourceType/_default`,
```

Comment:
Sorry, my earlier comment was a little too short to convey my meaning correctly: what I meant that the default provider could be read and set at `/:resourceType/_providers/_default` so that under `:resourceType` there is only one special path and _default_ is always associated with _provider_, otherwise it kinda looks like _the default resource_, not provider.

What do you think?

Reply:
I have no problem with that and will update it.